common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,4 +1,4 @@
-@(file: String, path: String = "", classes: Seq[String] = List())
-<span class="inline-@file @if(path){inline-@path} @classes.mkString(" ")">
+@(file: String, path: String = "", classes: Seq[String] = List(), title: Option[String] = None)
+<span class="inline-@file @if(path){inline-@path} @classes.mkString(" ")" title="@title">
```

Comment:
this shoudl be in a conditional, like path - i.e. if none is supplied, don't render it